pkg/controller/deployment/deployment_controller.go

```diff
@@ -78,7 +78,11 @@ type DeploymentController struct {
 	podStoreSynced func() bool
 
 	// A TTLCache of pod creates/deletes each deployment expects to see
-	expectations controller.ControllerExpectationsInterface
+	podExpectations controller.ControllerExpectationsInterface
+
+	// A TTLCache of rc creates/deletes each deployment expects to see
+	// TODO: make expectation model understand (rc) updates (besides adds and deletes)
+	rcExpectations controller.ControllerExpectationsInterface
```

Comment:
suggest also renaming the variables in controller_utils that indicate that expectations are only used for pods